tox.ini

```diff
@@ -1,13 +1,22 @@
 [tox]
 # note that tox expects interpreters to be found at C:\PythonXY,
 # with XY being python version ("27" or "34") for instance
-envlist = py26, py27, py32, py33, py34, docs
+envlist = py{27,34}-pyqt4, py34-pyqt5, py{26,27,32,33,34}-pyside, docs
+
+
 [testenv]
-deps=pytest 
-    pyside    
+deps=pytest
+    py{26,27,32,33,34}-pyside: pyside
 commands=py.test {posargs}
+setenv=
+    py{26,27,32,33,34}-pyside: PYTEST_QT_API = pyside
+    py{26,27,32,33,34}-pyqt4: PYTEST_QT_API = pyqt4
+    py{26,27,32,33,34}-pyqt5: PYTEST_QT_API = pyqt5
+sitepackages=True
+passenv=QT_QPA_PLATFORM_PLUGIN_PATH
```

Comment:
Why not use `setenv` for `QT_QPA_PLATFORM_PLUGIN_PATH` instead of telling the user to do it?

That's what I have in qutebrowser's `tox.ini`:

```
# https://bitbucket.org/hpk42/tox/issue/246/ - only needed for Windows though
setenv =
    QT_QPA_PLATFORM_PLUGIN_PATH={envdir}/Lib/site-packages/PyQt5/plugins/platforms
```